$MirOS: ports/misc/mc/patches/patch-src_user_h,v 1.3 2006/02/21 13:39:42 tg Exp $
--- src/user.h.orig	Fri Nov 29 03:03:31 2002
+++ src/user.h	Tue Feb 21 14:04:14 2006
@@ -21,10 +21,10 @@ int check_format_cd (const char *);
 #else
 #   define CEDIT_GLOBAL_MENU    "cedit.menu"
 #   define CEDIT_LOCAL_MENU     ".cedit.menu"
-#   define CEDIT_HOME_MENU      ".mc/cedit/menu"
+#   define CEDIT_HOME_MENU      ".etc/mc/cedit/menu"
 #   define MC_GLOBAL_MENU       "mc.menu"
 #   define MC_LOCAL_MENU        ".mc.menu"
-#   define MC_HOME_MENU         ".mc/menu"
+#   define MC_HOME_MENU         ".etc/mc/menu"
 #   define MC_HINT              "mc.hint"
 #endif
 
